vite.config.ts

```
@@ -15,6 +15,13 @@ export default defineConfig({
   ],
   server: {
     port: 3000,
+    proxy: {
+      "/api": {
+        target: "http://juhoec2ddns.ddns.net:8080",
+        changeOrigin: true,
+        rewrite: (path) => path.replace(/^\/api/, ""),
```

ESLint

[error] Replace `(path)` with `path`

(prettier/prettier)